diff --git syntaxes/02_Extra/cmd-help/syntaxes/cmd-help.sublime-syntax syntaxes/02_Extra/cmd-help/syntaxes/cmd-help.sublime-syntax
index ba80f9e..abba2c2 100644
--- syntaxes/02_Extra/cmd-help/syntaxes/cmd-help.sublime-syntax	
+++ syntaxes/02_Extra/cmd-help/syntaxes/cmd-help.sublime-syntax	
@@ -89,7 +89,7 @@ contexts:
     # abort if color escape codes
     - include: color-escape-codes-sink
     # abort if formatted with backspaces
-    - match: (?=.*[\b])
+    - match: (?=.*\b)
       set: sink
 
   main: # entry point only, main-b is the real main
