diff --git syntaxes/02_Extra/SCSS_Sass/Syntaxes/Sass.sublime-syntax syntaxes/02_Extra/SCSS_Sass/Syntaxes/Sass.sublime-syntax
index b181092..4479fbb 100644
--- syntaxes/02_Extra/SCSS_Sass/Syntaxes/Sass.sublime-syntax	
+++ syntaxes/02_Extra/SCSS_Sass/Syntaxes/Sass.sublime-syntax	
@@ -22,7 +22,7 @@ variables:
   # 2. Breaks are ascii characters other than denoted by {{nmchar}}.
   # 3. Assume unicode or escape character if backslash is matched, instead of
   #    matching (?!{{escape}}) which also effects performance negative.
-  break: (?=[[^-_[:alnum:]\\]&&[[:ascii:]]]|\Z)
+  break: (?=[[^-_[:alnum:]\\]&&[[:ascii:]]]|$)
 
   integer: '([-+]?)(\d+)'
   float: '([-+]?)(\d*(\.)\d+(?:[eE][-+]?\d+)?|\d+[eE][-+]?\d+)'
