diff --git syntaxes/02_Extra/VimHelp.sublime-syntax syntaxes/02_Extra/VimHelp.sublime-syntax
index 727a3ca..c11314c 100644
--- syntaxes/02_Extra/VimHelp.sublime-syntax	
+++ syntaxes/02_Extra/VimHelp.sublime-syntax	
@@ -9,7 +9,7 @@ file_extensions:
 # $VIMRUNTIME/syntax/help.vim
 contexts:
   main:
-    - match: '(?<=^\s*)(vim?|ex):\s*([a-z]+(=[^\s:]+)?(\s+|:))+'
+    - match: '(?:^\s*)(vim?|ex):\s*([a-z]+(=[^\s:]+)?(\s+|:))+'
       scope: comment.line.modeline.vimhelp
     - match: '^[-A-Z .][-A-Z0-9 .()_]*(?=\s+\*|$)'
       scope: markup.heading.headline.vimhelp
@@ -67,7 +67,7 @@ contexts:
         1: punctuation.definition.link.command.begin.vimhelp
         2: markup.underline.link.command.vimhelp
         3: punctuation.definition.link.command.end.vimhelp
-    - match: '(?<=\s*)(.*?)(?=\s?)(~)$'
+    - match: '(?:\s*)(.*?)(?=\s?)(~)$'
       captures:
         1: markup.heading.header.vimhelp
         2: punctuation.definition.keyword.vimhelp
